test/data/test_builtin_datasets.py

```diff
@@ -31,8 +34,28 @@ def test_wikitext2(self):
             datafile = os.path.join(self.project_root, ".data", "wikitext-2")
             conditional_remove(datafile)
 
+    def test_wikitext2(self):
+        from torchtext.datasets import WikiText2
+        # smoke test to ensure wikitext2 works properly
+        train_dataset, test_dataset, valid_dataset = WikiText2()
+        self.assertEqual(len(train_dataset), 1947375)
+        self.assertEqual(len(test_dataset), 230357)
+        self.assertEqual(len(valid_dataset), 203947)
+
+        vocab = train_dataset.get_vocab()
+        tokens_ids = [vocab[token] for token in 'the player characters rest'.split()]
+        self.assertEqual(tokens_ids, [2, 285, 502, 699])
+
+        # Delete the dataset after we're done to save disk space on CI
+        if os.environ.get("TRAVIS") == "true":
```

Comment:
This seems like a generic cleanup pattern. You should use the capabilities the test framework brings along for this.